Python/Algorithms/data_structures/linked_list_structs/linked_list.py

```diff
@@ -155,7 +164,11 @@ def insert(self, value):
         SingleLinkedListCommon.push_front(self, value)
 
     def remove(self):
+        if self.empty():
+            raise Exception('Empty LIFO strcture')
+        value = self.root.value
```

Comment:
There was a bit of an issue here. The current implementation of the LIFO and FIFO classes don't return any value when we call `remove()`. To return those value I added those exceptions. It may or may not return any value when being used in `LinkedList`.

Reply:
Then handle it at one place. Always remember the principle of inheritence. BY the way, this is not a structure. There are no concept of structures in python. So **remove structs completely**

Reply:
I have updated the handling for empty lists in the common methods.